domain/domain.go

```diff
@@ -460,18 +461,18 @@ func (do *Domain) loadSchemaInLoop(lease time.Duration) {
 			do.SchemaValidator.Stop()
 			err := do.mustRestartSyncer()
 			if err != nil {
-				log.Errorf("[ddl] reload schema in loop, schema syncer restart err %v", errors.ErrorStack(err))
+				logutil.Logger(context.Background()).Error("reload schema in loop, schema syncer restart err.", zap.Error(err))
```

Comment:
s/reload schema in loop, schema syncer restart err./reload schema in loop, schema syncer restart failed

Reply:
```suggestion
				logutil.Logger(context.Background()).Error("reload schema in loop, schema syncer restart failed.", zap.Error(err))
```